### PR TITLE
Bug Fixes for #124 and #125

### PR DESCRIPTION
**Pull request by Nischay-Pro**
Renamed Some files due to Build Error

Bug Fixes for #124 and #125 in m layout.

### CODE REVIEW

**Comment by Nischay-Pro** [comment]
It's not working in openSUSE. It's case sensitive I believe.

File: www/m/js/layout/RadareInfiniteBlock.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import {NavigatorDirection} from '../core/NavigatorDirection';
-import {InfiniteScrolling} from '../helpers/InfiniteScrolling';
+import {InfiniteScrolling} from '../helpers/infiniteScrolling';
```
Comment (radare) [inline www/m/js/layout/RadareInfiniteBlock.js]: Isnt this a class? Arent we using uppercase here?

File: www/m/js/modules/disasm/Disassembly.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import {DisassemblyNavigator} from './DisassemblyNavigator';
 import {RadareInfiniteBlock} from '../../layout/RadareInfiniteBlock';
-import {FlexContainer} from '../../layout/FlexContainer';
+import {FlexContainer} from '../../layout/flexContainer';
```
Comment (radare) [inline www/m/js/modules/disasm/Disassembly.js]: Same here

File: www/m/js/modules/disasm/Disassembly.js [hunk 3]
```diff
@@ -153,6 +153,16 @@ export class Disassembly extends RadareInfiniteBlock {
 				}
 			}
 		},{
+			name: 'Analyse reference',
+			ugly: 'ref',
+			active: false,
+			action: function(active) {
+				if (!active) {
```
Comment (radare) [inline www/m/js/modules/disasm/Disassembly.js]: Swap this condition and remove the return statement

**Comment by Nischay-Pro** [comment]
I'll make the necessary changes and push again.

**Comment by radare** [comment]
??? You can update the pr no need to close it

File: www/m/js/layout/RadareInfiniteBlock.js (hunk 1 above)
Comment (Nischay-Pro) [inline www/m/js/layout/RadareInfiniteBlock.js]: We are importing the class from a file. The file name can be anything even ../helpers/abc
import {InfiniteScrolling} from '../helpers/abc';

Class Name is independent of Filename. Windows I think ignores case sensitivity cause @gcolajan had no issues in Windows, whereas I had when using Windows.

I'll rename the files to reflect their Internal Class Names so we don't have this issue again.

**Comment by Nischay-Pro** [comment]
Bug Fix for Issue #123 
Also there doesn't exist a "io.debug" configuration variable. The console just throws up and error.

**Comment by Nischay-Pro** [comment]
Bug Fix for Issue #123 
Also, there doesn't exist an "io.debug" configuration variable. The console just throws up and error.

**Comment by Nischay-Pro** [comment]
Bug Fix for Issue #141

File: www/m/js/modules/disasm/Disassembly.js [hunk 4]
```diff
@@ -625,6 +632,7 @@ export class Disassembly extends RadareInfiniteBlock {
 			// { name: 'define flag color (fc)', shortcut: 'C', fct: function(evt, offset) { return applyOp(offset, 'C'); } },
 			// { name: 'set as data', shortcut: 'd', fct: function(evt, offset) { return applyOp(offset, 'd'); } },
 			// { name: 'end of function', shortcut: 'e', fct: function(evt, offset) { return applyOp(offset, 'e'); } },
+			{ aoj: false, name: 'view graph', shortcut: 'G', fct: function loadgraph() { return uiContext.navigateTo(Widgets.DISASSEMBLY_GRAPH);}},
```
Comment (gcolajan) [inline www/m/js/modules/disasm/Disassembly.js]: Anonymous functions is better here, or even arrow functions: `() => uiContext.navigateTo(...)`. No need to return.

You can refactor this part if you want.

**Pull request merged**